src/QMCWaveFunctions/Fermion/MultiDiracDeterminant.h

```diff
@@ -582,6 +592,9 @@ class MultiDiracDeterminant : public WaveFunctionComponent
   std::shared_ptr<std::vector<int>> detData;
   std::shared_ptr<std::vector<std::pair<int, int>>> uniquePairs;
   std::shared_ptr<std::vector<RealType>> DetSigns;
+  /* number of unique determinants at each excitation level (relative to reference)
+   * {1, n_singles, n_doubles, n_triples, ...}
```

Comment:
double asteroids on the first commend line.

Reply:
fixed